chainer/training/extensions/plot_report.py

```diff
@@ -23,28 +24,28 @@
 
 def _check_available():
     if not _available:
-        msg = '''matplotlib is not installed on your environment.
-Please install matplotlib to plot figure.
-
-  $ pip install matplotlib'''
-        raise RuntimeError(msg)
+        warnings.warn('matplotlib is not installed on your environment, '
+                      'so that nothing will be plotted at this time. '
+                      'Please install matplotlib to plot figures.\n\n'
+                      '$ pip install matplotlib')
```

Comment:
(Minor comment) Adding two spaces at the first position and \n at the end of the line may make the message more readable.